schemas/system/Schema.schema

```diff
@@ -1,7 +1,7 @@
 System({
   Category: { category: 'Category', lookup: true, required: true, index: true },
   Version: { domain: 'Version', required: true, index: true },
-  Schema: { domain: 'JSON', required: true },
+  Schema: { domain: 'Metaschema', required: true },
```

Comment:
Just `Schema`

Reply:
@tshemsedinov what about `SchemaSource`? It will help to distinguish the actual parsed schema from its source that is stored in this field.

Reply:
Something like `Source` or `Script`